### PR TITLE
Update to gradle 8.1, enable windows arm64 builds

### PR DESCRIPTION
**Pull request by ThadHouse**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -65,7 +65,7 @@ jobs:
           java-version: 11
           architecture: ${{ matrix.architecture }}
       - name: Build with Gradle
-        run: ./gradlew build
```
Comment (sciencewhiz) [inline .github/workflows/ci.yml:68]: Is this change intentional?

Comment (ThadHouse) [reply .github/workflows/ci.yml:68]: Yes. This will enable the arm64 builds. The flag is ignored everywhere except the x64 windows build. So no need to if out this.

**Review by PeterJohnson — APPROVED** [review]
